src/LCT.PackageIdentifier/NugetDevDependencyParser.cs

```diff
@@ -251,7 +252,15 @@ private static void GetDependencies(LockFileTargetLibrary library, NuGetComponen
                 }
                 else
                 {
-                    components.Add(depPackage.Id, depPackage);
+                    if (depPackage.Version.StartsWith(Dataconstant.OpenSquareBracket) && depPackage.Version.EndsWith(Dataconstant.CloseBracket))
+                    {
+                        //Do nothing
```

Comment:
your getting the same component count before and after doing this changes ?

Reply:
Now its same.I make some code changes accordingly